app/views/users/update.js.erb

```diff
@@ -0,0 +1,17 @@
+var emailButton = document.getElementById('emailButton');
+var notificationElement = document.getElementById('notifiID');
+
+if (emailButton.innerHTML === "Email Notifications On"){
+	emailButton.innerHTML = "Email Notifications Off";
+}else{
+	emailButton.innerHTML = "Email Notifications On";
+}
+
+if(notificationElement){
+	notificationElement.innerHTML = "<%= j render partial: 'shared/flashes', locals: {notice: flash[:notice]} %>";
+}else{
```

Comment:
I think there should be some spaces around `if` and `else`? not sure abotu JS code style best practices